packages/desktop-client/src/components/reports/graphs/NetWorthGraph.tsx

```diff
@@ -17,12 +17,20 @@ import Tooltip from '../Tooltip';
 
 import { Area } from './common';
 
+type domainProp = [number, number];
+
 type NetWorthGraphProps = {
   style?: CSSProperties;
   graphData;
   compact: boolean;
+  domain: domainProp;
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏`[number, number]` won't be correct here. You could either do `{ y?: [number, number]; x?: [number, number] }` or..

```suggestion
  domain: React.ComponentProps<typeof Chart>["domain"];
```